src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+                            json.
+        --query            : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose          : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        List container registries and show result in a table
+            az acr list -o table
+        List container registries in a resource group and show result in a table
+            az acr list -g <resource-group> -o table
+
+Create a container registry
+-------------
+::
+
+    Command
+        az acr create: Create a container registry.
+
+    Arguments
+        --location -l       [Required]: Location.
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group.
+        --storage-account-key -k      : Key of storage account.
+        --storage-account-name -s     : Name of storage account.
+
+    Global Arguments
+        --debug                       : Increase logging verbosity to show all debug logs.
+        --help -h                     : Show this help message and exit.
+        --output -o                   : Output format.  Allowed values: json, jsonc, list, table, tsv.
+                                        Default: json.
+        --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                        information and examples.
+        --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Create a container registry with managed storage account
+            az acr create -n <registry-name> -g <resource-group> -l <location>
+        Create a container registry with new/existing storage account in the current subscription
+            az acr create -n <registry-name> -g <resource-group> -l <location> -s <storage-account-name>
+        Create a container registry with your own storage account
+            az acr create -n <registry-name> -g <resource-group> -l <location> -s <storage-account-name>
+            -k <storage-account-key>
+
+Delete a container registry
+-------------
+::
+
+    Command
+        az acr delete: Delete a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Delete a container registry
+            az acr delete -n <registry-name>
+
```

Comment:
I'm not sure we need examples for every command, especially when there is only way thing you can do and only one way to do it. @johanste, thoughts?